bin/flush_cache

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+require_relative '../deployment'
+require 'cdo/aws/cloudfront'
+
+require 'sshkit'
+require 'sshkit/dsl'
+include SSHKit::DSL
+
+CDO.log.info 'Flushing Varnish caches on frontends...'
+on CDO.app_servers.values do |host|
+  execute "sudo varnishadm -T localhost:6082 -S /etc/varnish/secret 'ban obj.status ~ .'"
+end
+CDO.log.info 'Flushing CloudFront caches (up to 15 min)...'
+AWS::CloudFront.invalidate_caches
```

Comment:
Is there a potential race condition here, where the Cloudfront cache might invalidate before the Varnish flush is complete? (in which case we could end up re-caching old stuff at the Cloudfront level before the Varnish cache is refreshed)

Or does the sshkit call block until those invalidations are complete?